content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -99,10 +99,12 @@ limitation and compatibility rules will change.
 
 #### Pause Image
 
-Microsoft maintains a Windows pause infrastructure container at
-`mcr.microsoft.com/oss/kubernetes/pause:3.4.1`.
-Kubernetes maintains a multi-architecture image `k8s.gcr.io/pause:3.5` that
-supports Linux as well as Windows.
+Kubernetes maintains a multi-architecture image that includes support for Windows at `k8s.gcr.io/pause:3.5`.
```

Comment:
```suggestion
Kubernetes maintains a multi-architecture image as `k8s.gcr.io/pause`. From
version 3.5 (image tag ``k8s.gcr.io/pause:3.5`), that image includes support for Windows.
```
?

Reply:
I do want to be explicit about the version we are asking people to use to make sure the pause image contains images for each Windows OS version listed at https://kubernetes.io/docs/setup/production-environment/windows/intro-windows-in-kubernetes/#windows-os-version-support
 
In the past we've seen a lot of users run into issues where they were using an old pause image on newer OS releases so I would like to just give very prescriptive guidance here.

Reply:
@jsturtevant do you have any thoughts here?

Reply:
How about “For Kubernetes v1.22, the recommended `pause` image is xxxx”. Then, every release, track a task to make sure to review that the advice is current.

(If you don't want that work every release, then instead I recommend a fuzzier wording that leaves the onus on the reader to check. What I think none of is want is a page that isn't maintained and ends up being outright inaccurate.)

Reply:
That works!
We are committed to updating the supporting Windows OS versions every release and I see making sure the docs specify an appropriate pause image that works with all of the supported Windows OS versions as part of that support.